src/scss/components/_listings.scss

```diff
@@ -164,6 +164,56 @@ $listing-promoIconColor         : $orange;
             }
         }
 
+        .c-listing-item-badge {
+
+            /*
+            The below styles are in place to juggle the badge items between stacked and 
+            inline and targets viewports inbetween mid and wide (landscape mobiles and very small desktops).
+
+            On narrow, mid and wide you will see:
+            
+            item • item
+            
+            between mid and wide you will see:
+
+            item
+            item
+
+            The mixins are used to stop code duplication as each chunk of 
+            code needs to be within its own media mixin.
+            */
+
+            @mixin inlineSpan() {
```

Comment:
Only thing that would be better here is if these were named less tied to element names (as the span's may one day change in the future and then the name of the mixins would need updating too).

Looks good otherwise.